core/src/main/kotlin/io/specmatic/core/pattern/JSONObjectPattern.kt

```diff
@@ -582,15 +588,10 @@ data class JSONObjectPattern(
         
         return if (anyPatternPaths.isNotEmpty()) {
             anyPatternPaths.map { anyPath ->
-                // Format the anyPath similar to calculatePathForAnyPattern
-                val formattedPath = if (anyPath.matches("^[a-zA-Z][a-zA-Z0-9]*$".toRegex())) {
-                    // It's a typeAlias, wrap in braces
-                    "{$anyPath}"
-                } else if (anyPath in setOf("string", "number", "boolean")) {
-                    // It's a scalar type, wrap in braces
+                // Format the anyPath using the same logic as calculatePathForAnyPattern
```

Comment:
isn't this logic comparable to `private fun calculatePathForJSONObjectPattern(key: String, childValue: Value, objectPattern: JSONObjectPattern, resolver: Resolver): List<String> {`?